packages/toast/src/ControlledToast/ControlledToast.story.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import { ComponentStory } from '@storybook/react';
+
+import Button from '@leafygreen-ui/button';
+import { css } from '@leafygreen-ui/emotion';
+import { StoryMeta } from '@leafygreen-ui/lib';
+
+import { ToastProvider, Variant } from '..';
+
+import { ControlledToast as Toast } from './ControlledToast';
+
+export default StoryMeta({
+  title: 'Components/Toast/Controlled',
+  component: Toast,
+  parameters: {
+    default: 'Basic',
+    controls: {
+      exclude: ['open', 'onClose'],
+    },
+  },
+  args: {
+    title: 'Velit ea exercitation qui aute dolor proident.',
+    description: 'Exercitation incididunt ea proident velit mollit',
+    open: true,
+    variant: Variant.Note,
+    progress: 0,
+    className: css`
+      z-index: 1;
+    `,
+  },
+  argTypes: {
+    variant: {
+      control: 'select',
+      options: Object.values(Variant),
+      default: Variant.Note,
+    },
+    progress: {
+      control: { type: 'range', min: 0, max: 1, step: 0.1 },
+      if: { arg: 'variant', eq: Variant.Progress },
+    },
+    description: {
+      control: 'text',
+    },
+    title: {
+      control: 'text',
+    },
+  },
+});
+
+export const Basic: ComponentStory<typeof Toast> = args => {
+  const [open, setOpen] = useState(false);
+
+  return (
+    <ToastProvider>
+      <Button onClick={() => setOpen(!open)}>
+        {open ? 'Close' : 'Open'} Toast
+      </Button>
+      <Toast
+        {...args}
+        open={open}
+        onClose={() => {
+          setOpen(false);
+        }}
+      />
+    </ToastProvider>
+  );
+};
```

Comment:
nit: since there's only one story in here, this could probably just live in `Toast.tsx` without needing a story subdir

Reply:
I left it here since the StoryMeta is quite different, and we may want to add more stories in the future

Reply:
We _could_ move the file up one dir level, but I feel like it makes more sense here with the specific component files

Reply:
Yeah it's fine if you want to keep it - I only call it out because it's not really best practice. I believe Storybook only considers different configurations of different metas necessary when the story is about a new component altogether (so they can generate the right controls for it), not for when the arguments need to be different, even if they are quite different. Story-specific configurations should really be done through `ControlledToast.args` or `ControlledToast.argTypes`

Reply:
It is _technically_ a different component (`ControlledToast`)